client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -54,6 +59,10 @@ import type { TranslateResult } from 'i18n-calypso';
 // This will make converting to TS less noisy. The order of components can be reorganized later
 /* eslint-disable @typescript-eslint/no-use-before-define */
 
+function isPlanEligibleForCheckoutFeatureListExperiment( plan: ResponseCartProduct ) {
+	return isPersonal( plan ) || isPremium( plan ) || isBusiness( plan ) || isEcommerce( plan );
```

Comment:
This function is temporary and will be removed after the experiment is concluded. If we ship `treatment`, `getPlanFeatures` will most likely just check if the plan object has a `getCheckoutFeatures` function, otherwise, it will fall back to the present values.